app/src/main/java/com/endava/myendava/adapters/EventParentAdapter.java

```diff
@@ -0,0 +1,70 @@
+package com.endava.myendava.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.endava.myendava.R;
+import com.endava.myendava.models.EventParent;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class EventParentAdapter extends RecyclerView.Adapter<EventParentAdapter.ProjectParentViewHolder> {
```

Comment:
no need for this adapter ... please see https://medium.com/@saber.solooki/sticky-header-for-recyclerview-c0eb551c3f68